### DISCUSSION TITLE
[FR] speaker group in media_player type

### DISCUSSION THREAD

**Discussion by saya6k** [Clooos/Bubble-Card#567] [Feature requests]
Lovelace card like mini media player support [speaker group features](https://github.com/kalkih/mini-media-player#speaker-group-management)

It would be great if bubble card media player type also support speaker group features like join/unjoin, reducing need to customize

Starting from some famous integration like Sonos or linkplay would be great idea.

**Comment by ayostepht** [comment]
Second this! Mini Media Player has implemented this well (https://github.com/kalkih/mini-media-player/)

**Comment by blueJack92** [comment]
That would be a really great feature! HA has now officially added the feature: https://rc.home-assistant.io/blog/2025/05/26/release-20256/#joinunjoin-groups-of-media-players

**Reply by Clooos** [reply]
Hi! This would be great indeed, but I haven't any speaker group to test that 😅

**Reply by blueJack92** [reply]
Of course I would test it! 😁
